src/app/partials/nav/header-static/header-static.component.scss

```diff
@@ -3,19 +3,299 @@
+		opacity: 0;
+	}
+}
+
+.menu-change {
+	.menu-bar3 {
+	    -webkit-transform: rotate(45deg) translate(-8px, -8px);
+    	transform: rotate(45deg) translate(-8px, -8px);
+    }
+}
+
+.header-shadow{
+	@include shadow-btn;
+}
+
+.header-shadow-large{
+	@include shadow-header-large;
+}
+
+.dark-text{
+	color:$gray-darker !important;
+}
+
+.invert-color{
+	filter: invert(100%);
+}
+
+.logo-container {
+	text-align: right;
+	display:inline-block;
+	float: left;
+	width:10%;
+	margin-top:30px;
+	margin-left:30px;
+	img:hover {
+		cursor:pointer;
+	}
+	img {
+		height:40px;
+	}
+}
+
+.navlinks-container {
+	display:inline-block;
+	float: right;
+	width:70%;
+	position:relative;
+	text-align:right;
+	transition:height 0.5s ease;
+	padding-bottom:10px;
+	padding-right:10px;
+	div{
+		display:block;
+		position:relative;
+		margin-top:20px;
+		padding-top:10px;
+		height:100%;
+		margin-right:20px;
+	}
+	ul {
+		list-style: none;
+		margin-top:30px;
+		li {
+			display: inline;
+		}
+	}
+}
+
+.btn-nofill{
+    padding:10px 30px 10px 30px;
+    border:1.5px solid white;
+    border-radius:20px;
+    margin-left:5px;
+    margin-bottom:5px;
+    color:white;
+    text-align: center;
+    cursor:pointer;
+    @include shadow-btn;
+    &:hover {
+	  background: $gray-light;
+	  color: black;
+	  transition:all 0.5s ease;
+	  @include shadow-btn-large;
+	}
+}
+
+.btn-fire {
+	display:inline-block;
+	@include fire(false);
+    border: none;
+    color: white;
+    padding:13px 33px 13px 33px;
+ 	border-radius:20px;
+    margin-left:5px;
+    margin-bottom:5px;
+    margin-top:0 !important;
+    margin-bottom: 0 !important;
+    text-align: center;
+    cursor:pointer;
+    @include shadow-btn;
+    &:hover {
+	@include fire(true);
+		@include shadow-btn-large;
+	 	transition:all 0.5s ease;
+	}
+}
+
+.btn-fill {
+    padding:13px 33px 13px 33px;
+    border-radius:20px;
+    margin-left:5px;
+    margin-bottom:5px;
+    background:white;
+    color:$blue-dark;
+    text-align: center;
+    cursor:pointer;
+    @include shadow-btn;
+    &:hover {
+    	@include shadow-btn-large;
+		background: $gray-light;
+		color: black;
+		transition:all 0.5s ease;
+	}
+}
+
+.btn-white-bg {
+	border:1.5px solid white;
+}
+
+.navlinks {
+	font-size:$fs-16;
+	text-align:right;
+	cursor:pointer;
+	margin-top:10px;
+	img {
+		display: inline-block;
+		vertical-align:middle;
+		line-height:50px;
+		height:40px;
+		transform: rotate(90deg);
+		text-align: center;
+		margin-top:10x;
+	}
+}
+.navlinktext {
+	margin:20px;
+	color:white;
+}
+
+@media(min-width:$screen-sm) and (max-width:$screen-md){
+
+	.navlinks-container{
+		width:100%;
+		text-align:center;
+		div{
+			margin-right:0;
+		}
+	}
+
+	.logo-container {
+		width:100%;
+		text-align:center;
+		margin-top:30px;
+		margin-left:0;
+	}
+}
+@include screen-small {
+
+	.menu-expander{
+		display:inline-block;
+	}
+
+	.header-body-buttons{
+		width:100%;
+	}
+
+	header {
+		.btn-fill, .btn-nofill, .btn-fire{
+			max-width: 50%;
```

Comment:
Indentation @live-wire

Reply:
@live-wire if you are using Sublime editor then please install html-css prettify for indentation and change setting to `space 2`

Reply:
Yes, I'm using sublime. Thanks for the tip! 👍